gulpfile.js

```diff
@@ -2,13 +2,29 @@ var copy = require("copy"),
     clear = require("cli-clear"),
     del = require("del"),
     gulp = require("gulp"),
-    gutil = require("gulp-util");
+    runSequence = require("run-sequence"),
+    sourcemaps = require("gulp-sourcemaps"),
+    ts = require("gulp-typescript"),
+    tsProject = ts.createProject("../../tsconfig.json");
 
 gulp.task("clear", function(cb) {
     clear();
     cb();
 });
 
+gulp.task("clean", function() {
+    return del("./coverage");
+});
+
+gulp.task("build", function() {
+    var tsResult = tsProject.src()
+        .pipe(sourcemaps.init())
+        .pipe(tsProject());
+    return tsResult.js
```

Comment:
Would it be nicer to:
```
tsProject.src()
    .pipe(sourcemaps.init())
    .pipe(tsProject()).js
    .pipe(sourcemaps.write("."))
    .pipe(gulp.dest(...));
```
?

I'm assuming that you're following the example [here](https://github.com/ivogabe/gulp-typescript#source-maps), which also works! Just a suggestion to make it cleaner :)